community-operators/triggermesh/triggermesh.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,167 @@
rJmkNwO2LEzOlpmU+SYNvZ/Oe1gSfa9NkBszebbPrrbLLieGsETs3Gde22lSGF9uW9+AEQ2vJgczYEdRZ3JR2RQcXWOQEMI4B0Yj/18A1kBwW6vS8IyocZkFqBOuKN+AydXkI911GARTYH9sUNZ4kshzTzde+vSulN2Ev/rq9GJFsY+rYJRtxaCE6J58CTiXMOvO5/mpfgmd+VuR+F4+Mh9jpu7jlLZt4KrmxQgX5qoZSnbfBc38idzCgVkDWRKV+kd607YxqVsFp0Y9thmwXexr65iWLWhpbdFyQsDMoq42GqoW3DQg69vTR3RHf8st1kwUcSlbmmC6ZGxQHjUh4Jcgaavst8hQWp4D9QF75PcNzJM+hZiIjmdKQ08OvrjiRK46gR5YTBhbxK1uaYnhphcO1AdsQyHfSyvqKESuli4cW0BNBW0C1wamP/XUA6uNmLZLSLpYcu+k3upYNHxokrY+YHuWmUwa4zVNcW/NWcNybYKcNJH2w7qHABNMLp0KQXLAwrBJpk/b6pew0BDY0Hfah7W5fYDFvbX5VDh78iIrtrjlaQrc/qE0Loedf3Qageg6i1yzDIEdH+ZymkTMozzJHkSt2Mi20BzUSU6CW02aJDHJcZL/jiTd4h8jYSeaRWJgwhRDYJsVlq8wupaPa3iM8obUzlBJC0U3lwmcNNHZcsTwemT7Gc4QNfeXbH8cd63a8U3LI7Dj+JFV5a8TY/ClrM9exzT6RNGqYzRsKWAg7xq+OYXBZFacA2/KXIq51jVP+ZQ6tW1OMnhoaDxm1mMrKyzz+sdTXp8LWVvK1UIn+9Knye54kyyv+DDoizofIJI8uZ2PPgExUuSbqE7NMAK7IhgGM2Crt8S1x8LnpnXXkJ7ZJCuzC/ZloajMd/M85JKnkwhu6S+kUJ1agIPoQcGNXLQZsJktrrRkCBqSBvZM7DW4gIKOEPWhP6an1qcxjy3iIcFNp5cnRQ0oVyaJLXPGMSXEEAt78ci4cIKAroFd/EIklUhbZJhXziJI0LRWxIi9RLQCtdH3QMYmfm389KGO1T+TVuSQRzRtrSLT61vTjFi1xPA+iCnmu45SS5txc2CzMj9r7HqVnTLqGq6LI5jjFjQzXP1j+9lDgc9b75gOVnTVa+PoTtjYgScHnPEA6bEh0lGv1tXjWmI9R7qCmmRwxb7F6nlzUYSFczW13VIe5sqGrvShXe6HOZvH/lp6q4LTdcFPngOo+ekzHIFbEWc4c6bKlHTjpcSgNm49leIcv5zzGPg8oojPw55+h1VHdsCWDzryIErggcXYGYCeTA3XsFk9SyXP7zgx8XVFjpNSr/MXnFRZakZZnOJgHTwrILdyNL+aymMtn4dLGApud2IZ7IDNwpUA37Ospyh5R77FHDhRM+Fj8ERQEwBNDVLGr0QcZ0Yx0CzdeaBSBV4y8+WTzwJfmip2Q1xMbSGCJLTbydhJLjWQ5ABgLYnydzyWK8s+2ez10lZ1SeVqgsW1Z0hKSY5D6K/fSS68Hf29eLAwPDrv8tC9tSVdkBzP0HEodzhwHNOky1PnAzyD8/noyTFuwGa5VQYCk+kqT6HTVqiBopsMV1ibZz8b/mnMO8Q3cPJT2Nxm3aAZg7P3MHtRJGk81VGsuPpgMinR/Lj0qp9LPGptIq6di09F0AMp92Xhz8QbKXqfS6ns1U/90wgT2IxIokTFfnViZ1xCHG9XUXmarnnUt5LgvvV4HTIfxZJSUOkLcImFqKKTueTeJ7G2QPquo0P2rHtf7kBA54yFXkeN1XLaavORddAYi2uoqD9eWXQTL2VnPG6OrqTOGsZjFYJ7j51USnCrJf+DJKreo0ZbIbUEepUet9O4/5v3c6AmjQS6une6mORsj1icsvxORRrTjiUzFXUz18kAPxc9FxFU7Xnwc1Id2OQW1YBKyN+tnXn0gJoNJd6BrkPsSG+nkc3Ke0xTGPz5FVkrqsOERu4CVhj0L1+/MH1zb2xLxz8e6PMD7ISQq+e2MVVNuXtKvfdxxSXegbJufhNS08fSNL72YxHRSrrO9Hl3ZfLuA/TqWhjiqK25N91noCHAImoITQocYla0okxV6BfYLJhqOPbecuUIVXchTOJA2RclnbcQ3LoXuynbdKcboD/vS7WlhYGTvpjxWP/ATsqnmSZdaEmAe+zBudlSLmjk7lya+Y+gs3lt0DknaryXAHR6xAZX9euUAdqGmkfWB+yEBglw2YNT0b6LT85hcsvpqHO3tlRcZiEYUpWXpqkyBZ6qo2wRauEmR6n8JZobsJT7Q+bCWi6mEREJoNHpKRv1Wqhyn6CpQo6a3OmgiAv48djGzzxI5zSZ5AXmoPx837/4fibx+OX6vS9FIbh9WfhxgEf7GE2gxmPSTrqlNOr5odd9auqvPwpglk5K4zvaGeMaCJgNsLMNkTYitJWWIEcPJAHPVAS8pufRzAiWTFVzT0ZqFbKyLDch2v7iK/3utIpGOsDZUKce/he8fCy5iEbeK933nBsuZWmc7ezjACQP8eME3iN5nIFteTOAjdZXDmq6+mlROdRnb/7kjEj2Z+ReiPt/foZtoku2taOGR2eTUlTJpPgJ253wxSPASSPPZ0LjpDbMyf3FATYZPgE4ls8kb4dhWUAuuY9VSOOF+qdxvPy5vap/8DhN1lCP7keHPsRsqj8RJOEBJ7JoA+En0Lrvsp+iFq+UxQI2n8/z1jr+U85zDcqqrI7VM6SImgBOG1cLisYa1GTVyGpO7sUDNgGpwL3rwOa+tHupe7Aj94tx3nhKgbpuGh2Y16QsiyVjZzmrVGy0futmb2WuBxARuDJ8PxNf76Wi7zoq6RhVxJ6eXqXq+poYETEfiRYb2MkzoMZkdKoDPeKK4MxlHLehU34kJ02asPpbAfwiyAWNOfVmH3H3IE/P/yp1NCSEwIHAgcCBwIHAgcCBwIHAgcCBwIHAgcCBwIHAgcCBwIHAgcCBwIHAgcCBwAEh/g+lrvnILbyNXAAAAABJRU5ErkJggg==
```

Comment:
This icon needs a `mediatype` field. I'm guessing this is a PNG; if so, use `mediatype: image/png`:
```
icon:
- base64data: BASE64HERE==
  mediatype: image/png
```

Reply:
fixed